datadog/resource_datadog_dashboard.go

```diff
@@ -4906,6 +4972,33 @@ func buildTerraformTimeseriesDefinition(datadogDefinition datadogV1.TimeseriesWi
 	return terraformDefinition
 }
 
+func getScatterplotFormulaSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"formula_expression": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "A string expression built from queries, formulas, and functions.",
+				},
+				"alias": {
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "An expression alias.",
+				},
+				"dimension": {
+					Description:      "Dimension of the Scatterplot.",
+					Type:             schema.TypeString,
+					Optional:         true,
```

Comment:
This is required according to [docs](https://docs.datadoghq.com/dashboards/widgets/scatter_plot/#api)

Reply:
Resolved in [e8e0](https://github.com/DataDog/terraform-provider-datadog/pull/1275/commits/9ecadcc1d889605b30f5de449d7098231ac2e8e0).